build.zig

```diff
@@ -17,7 +17,7 @@ pub fn build(b: *std.Build) !void {
     const lib_test = b.addTest(.{
         .target = target,
         .optimize = optimize,
-        .test_runner = b.path("test_runner.zig"),
+        .test_runner = .{ .path = b.path("test_runner.zig"), .mode = .simple },
```

Comment:
This change breaks my build when using Zig version `0.14.0-dev.2851+b074fb7dd` (latest master build from official Zig download page). The other changes in this PR are necessary.

Reply:
Looks like this is released now with ``0.14.0-dev.2987+183bb8b08``

Reply:
There's probably comptime way to support both methods.